src/main/java/io/strimzi/kafka/bridge/BridgeEndpoint.java

```diff
@@ -39,6 +39,13 @@
 	 * @param endpoint	Remote protocol endpoint to handle
 	 */
 	void handle(Endpoint<?> endpoint);
+
+    /**
+     * Handler for the remote protocol endpoint
+     * @param endpoint	Remote protocol endpoint to handle
+     * @param handler handler for result
+     */
+	void handle(Endpoint<?> endpoint, Handler<String> handler);
```

Comment:
the `String` is because we know that for the HTTP case we have a consumer-id but for example it doesn't make sense for the AMQP bridge. I know that we are not using this `handle` for the AMQP bridge but in any case we are forcing to an handler with `String` as return type. Can we be more generic like for the other `endpoint` parameter ?